modules/service-accounts-using-credentials-inside-a-container.adoc

```diff
@@ -26,15 +26,13 @@ by the token:
 +
 ----
 $ curl --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
-    "https://openshift.default.svc.cluster.local/oapi/v1/users/~" \
+    "https://openshift.default.svc.cluster.local/apis/v1/users/~" \
```

Comment:
@kalexand-rh Based on your work in https://github.com/openshift/openshift-docs/pull/13010/files, can you help me find the SME can suggest the correct replacement for this URL?

Reply:
@bergerhoffer, this looks like auth. Do you know?

Reply:
at first eye sight this looks correct to me. i'd like to verify on a cluster though.

Reply:
As far as I can tell from traversing through the API on an OpenShift 4.10 cluster it looks like so:

```bash
curl --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt "https://openshift.default.svc.cluster.local/apis/user.openhift.io/v1/users/~" -H "Authorization: Bearer $TOKEN"
```

and the output is like so:

```json
{
  "kind": "User",
  "apiVersion": "user.openshift.io/v1",
  "metadata": {
    "name": "system:serviceaccount:openshift-authentication:oauth-openshift",
    "creationTimestamp": null
  },
  "groups": [
    "system:authenticated",
    "system:serviceaccounts",
    "system:serviceaccounts:openshift-authentication"
  ]
}
```